### PR TITLE
Move Remixd To Monorepo

### PR DESCRIPTION
**Pull request by ioedeveloper**

### CODE REVIEW

File: libs/remixd/src/serviceList.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { RemixdClient as sharedfolder } from './services/remixdClient'
```
Comment (Aniket-Engg) [inline libs/remixd/src/serviceList.ts:1]: May be you can use `export { RemixdClient as sharedfolder } from './services/remixdClient'`

File: libs/remixd/package.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "remixd",
```
Comment (Aniket-Engg) [inline libs/remixd/package.json:2]: I think it should be `@remix-project/remixd`

File: libs/remixd/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+More details are explained in this [tutorial](http://remix.readthedocs.io/en/latest/tutorial_remixd_filesystem.html).
+
+Alternatively `remixd` can be used to setup a development environment that can be used with other popular frameworks like Embark, Truffle, Ganache, etc..
+
+`remixd` needs `npm` and `node`
+
+## INSTALLATION
+
+`npm install -g remixd`
```
Comment (Aniket-Engg) [inline libs/remixd/README.md:16]: May be `npm install -g @remix-project/remixd`

**Review by Aniket-Engg — APPROVED** [review]

**Comment by backinaaz-blip** [comment]
{ "extends": "../../.eslintrc", "rules": {}, "ignorePatterns": ["!**/*"] }chain-1-0x11111112-sources.zip

**Pull request merged**